### PR TITLE
EVG-18400 Fix bookmark causing search to jump

### PR DESCRIPTION
**Pull request by khelif96**
EVG-18400

### Description 
Fixes a bug that was causing scrollToLine to trigger whenever a bookmark was changed and a search was applied
Throws in a link to the wiki as well.

### Screenshots

https://user-images.githubusercontent.com/4605522/204659093-a56ce2b1-0cf0-4d9c-8c3f-9280c3838adf.mov

### CODE REVIEW

File: src/context/LogContext/index.tsx [hunk 1]
```diff
@@ -180,9 +180,6 @@ const LogContextProvider: React.FC<LogContextProviderProps> = ({
       type: "SET_MATCH_COUNT",
       matchCount: results.length,
     });
```
Comment (sophstad) [inline src/context/LogContext/index.tsx:182]: This is unrelated and I think has existed for a while, but uploading a file unexpectedly adds a bookmark to the n+1 line of the file.

**Review by sophstad — COMMENTED** [review]
I just tried testing this PR on Safari and noticed I'm unable to upload any files. I think this is the first time I've used Safari so this may not be a new issue. I don't use any extensions.

**Comment by khelif96** [comment]
> I just tried testing this PR on Safari and noticed I'm unable to upload any files... I think this is the first time I've used Safari so this may not be a new issue. And I don't use any extensions!

Strange was there any specific action you took? I'm unable to repro.

**Comment by sophstad** [comment]
> > I just tried testing this PR on Safari and noticed I'm unable to upload any files... I think this is the first time I've used Safari so this may not be a new issue. And I don't use any extensions!
> 
> Strange was there any specific action you took? I'm unable to repro.

Nothing in particular!

https://user-images.githubusercontent.com/9298431/204869410-5310412b-9617-444f-b6ce-45f6fa2212fb.mov

**Comment by khelif96** [comment]
> > > I just tried testing this PR on Safari and noticed I'm unable to upload any files... I think this is the first time I've used Safari so this may not be a new issue. And I don't use any extensions!
> > 
> > 
> > Strange was there any specific action you took? I'm unable to repro.
> 
> Nothing in particular!
> 
>  Screen.Recording.2022-11-30.at.12.39.44.PM.mov

https://jira.mongodb.org/browse/EVG-18414
I just noticed it only applies to the button not the drag and drop window

**Comment by khelif96** [comment]
> > > > I just tried testing this PR on Safari and noticed I'm unable to upload any files... I think this is the first time I've used Safari so this may not be a new issue. And I don't use any extensions!
> > > 
> > > 
> > > Strange was there any specific action you took? I'm unable to repro.
> > 
> > 
> > Nothing in particular!
> > Screen.Recording.2022-11-30.at.12.39.44.PM.mov
> 
> https://jira.mongodb.org/browse/EVG-18414 I just noticed it only applies to the button not the drag and drop window

Figured it out it just needed a e.stopPropogation() https://github.com/evergreen-ci/parsley/pull/111

File: src/context/LogContext/index.tsx (hunk 1 above)
Comment (khelif96) [reply src/context/LogContext/index.tsx:182]: There is a ticket for this. https://jira.mongodb.org/browse/EVG-18350

**Review by sophstad — APPROVED** [review]

**Pull request merged**